File: 1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int toMain = 0;
        for(auto& log : logs) {
            if(log == "../") {
                toMain = max(0, toMain - 1);
            } else if (log != "./") {
                toMain++;
            }
        }
        
        return toMain;
    }
};

File: 1598-crawler-log-folder/1598-crawler-log-folder.py
class Solution:
    def minOperations(self, logs: List[str]) -> int:
        toMain = 0
        for log in logs:
            if log == "../":
                toMain = max(0, toMain - 1)
            elif log != "./":
                toMain += 1
                
        return toMain;